queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name = 'mon';
+SELECT * animals WHERE date_of_birth = '2016 - 2019';
+SELECT * animals WHERE escape_attempts > 2 AND neutered = true;
+SELECT * animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT * from animals WHERE weight_kg > '10.0' name AND escape_attempts;
```

Comment:
- [ ] Kindly I think here also you have to select the `name, escape_attempts` not the whole data.